src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -507,6 +508,7 @@ void FrontEnd::convert(const std::shared_ptr<ov::Model>& partiallyConverted) con
 
 void FrontEnd::normalize(const std::shared_ptr<ov::Model>& model) const {
     ov::pass::Manager manager;
+    manager.register_pass<ov::pass::KeepFWPrecisionFor16BitFloatConstants>();
```

Comment:
can we have TF layer test for it? Or you can add TF FE unit-test with graph comparation?

Reply:
done. added for PT as well